src/platforms/unity/index.mdx

```diff
@@ -12,21 +16,99 @@ If you don't already have an account and Sentry project established, head over t
 
 ## Install
 
-Sentry captures data by using an SDK within your application’s runtime.
+Sentry captures data by using an SDK within your application’s runtime. 
+
+Install it from [`Unity UPM`](https://docs.unity3d.com/Manual/upm-ui.html) via [git url](https://docs.unity3d.com/Manual/upm-ui.html)
+
+```
+https://github.com/getsentry/sentry-unity.git?path=/package
```

Comment:
We'll get the version from: https://release-registry.services.sentry.io/sdks (still not up there for some reason).

The second argument is the fallback number. I set to 0.0.5 for now:

```suggestion
https://github.com/getsentry/sentry-unity.git?path=/package#{{ packages.version('sentry.dotnet.unity', '0.0.5') }}
```